doc/oil-help.md

```diff
@@ -143,7 +143,7 @@ Initializes a name to the Oil expression on the right.
     var s = 'mystr'        # equivalent to declare s=mystr
     var pat = / digit+ /   # an eggex, with no shell equivalent
 
-It's either a global or scoped to the current function.
+It's either global or scoped to the current function.
```

Comment:
I think "a global" was OK here... that is basically short for "a global variable".